model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -520,18 +504,15 @@ message ProcessBundleSplitResponse {
     // (Required) The grpc read transform reading this channel.
     string ptransform_id = 1;
 
-    // (Required) Name of the transform's input to which to pass the element.
-    string input_id = 2;
-
     // The last element of the input channel that should be entirely considered
     // part of the primary, identified by its absolute index in the (ordered)
     // channel.
-    int32 last_primary_element = 3;
+    int32 last_primary_element = 2;
 
     // The first element of the input channel that should be entirely considered
     // part of the residual, identified by its absolute index in the (ordered)
     // channel.
-    int32 first_residual_element = 4;
+    int32 first_residual_element = 3;
```

Comment:
I assume Portable pipelines aren't currently likely to be "updated" by users, which would make the proto re-numbering safe? That is, Portability is still notionally experimental, at least WRT work splitting?

Otherwise this could cause mismatches between an SDK with splitting, and a runner that supports splitting, and they wouldn't be able to communicate correctly.

Reply:
That is correct.
This is also renumbering a feature only used by SDF which doesn't exist anywhere.